connect-examples/v2/csharp_payment/Pages/ProcessPayment.cshtml.cs

```diff
@@ -1,39 +1,31 @@
 ﻿using System;
-
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Configuration;
 using Square.Connect.Api;
-using Square.Connect.Model;
 using Square.Connect.Client;
+using Square.Connect.Model;
 
-namespace CSharpPaymentExample
+namespace sqRazorSample.Pages
 {
-
-    public partial class Default : System.Web.UI.Page
+    public class ProcessPaymentModel : PageModel
     {
-        // The access token to use in all Connect API requests.
-        // Use your *sandbox* accesstoken if you're just testing things out.
-        private static string accessToken = "REPLACE_ME";
-
-        // The ID of the business location to associate processed payments with.
-        // See [Retrieve your business's locations]
-        // (https://docs.connect.squareup.com/articles/getting-started/#retrievemerchantprofile)
-        // for an easy way to get your business's location IDs.
-        // If you're testing things out, use a sandbox location ID.
-        public static string LocationId = "REPLACE_ME";
-        public static string ApplicationId = "REPLACE_ME";
-
-        static Default()
+        private readonly string LocationId;
+        
+        public string ResultMessage
         {
-            Configuration.Default.AccessToken = accessToken;
+            get;
+            set;
         }
 
-        protected void Page_Load(object sender, EventArgs e)
-        {
+        public ProcessPaymentModel(IConfiguration configuration) {
+            this.LocationId = configuration["AppSettings:LocationId"];
         }
 
-        [System.Web.Services.WebMethod]
-        public static string Charge(string nonce)
+        public void OnPost()
```

Comment:
I think it is a good practice to return a status code here based on response.

Reply:
I don't think this is necessary in razor page handler, the response code is handled internally based on the pagemodel methods invoked